summit/index.html

```diff
@@ -200,12 +197,11 @@ <h2>Who Should Come to The Impact Summit?</h2>
 					<div class="col-12 col-md-7">
 						<h2>Speakers and Workshop Leaders</h2>
 						<p class="lead">
-							Full 2019 speaker list coming soon.
-							<br/>
-							<a target="_blank" href="https://www.facebook.com/impactlabs.io">Follow us</a> for new speaker announcements!
+								The 2020 Impact Summit speakers have not yet been selected.
+								Follow us [hyperlinked to facebook group] for new speaker announcements!	
```

Comment:
quick change. this should hyperlink to https://www.facebook.com/impactlabs.io/. Good to merge after.